backend/run_context_tests.py

```
@@ -0,0 +1,73 @@
+    integration_passed, integration_failed = asyncio.run(run_integration_tests())
+    
+    # Summary
+    print()
+    print("=" * 60)
+    print("📊 FINAL TEST RESULTS")
+    print("=" * 60)
+    print(f"Unit Tests: {unit_passed} passed, {unit_failed} failed")
+    print(f"Integration Tests: {integration_passed} passed, {integration_failed} failed")
+    print(f"Total: {unit_passed + integration_passed} passed, {unit_failed + integration_failed} failed")
+    
+    total_passed = unit_passed + integration_passed
```

Ruff (0.12.2)

Local variable `total_passed` is assigned to but never used

Remove assignment to unused variable `total_passed`

(F841)